db/migrate/20230214100638_change_webhooks_organization_id_to_uuid.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ChangeWebhooksOrganizationIdToUuid < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :webhooks, :organization_id, :bigint
+
+    add_reference :webhooks, :organization, type: :uuid, index: true
```

Comment:
`null: false`?

Reply:
it should not, but if we have some users with `webhooks` already in database, it will break everything